Backend/Remora.Discord.Rest/API/Channels/DiscordRestChannelAPI.cs

```diff
@@ -80,6 +80,23 @@ public virtual Task<Result<IChannel>> GetChannelAsync
         );
     }
 
+    /// <inheritdoc />
+    public virtual Task<Result> SetVoiceChannelStatusAsync
+    (
+        Snowflake channelID,
+        Optional<string> status,
+        Optional<string> reason = default,
+        CancellationToken ct = default
+    )
+    {
+        return this.RestHttpClient.PatchAsync
```

Comment:
Missing `AddAuditReason` call

Reply:
This doesn't exist on the non-generic methods it seems. cc @Nihlus